src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -508,6 +506,10 @@ class ADIOSWriter : public ILightweightPlugin
             mpi_pos = gc.getPosition();
             mpi_size = gc.getGpuNodes();
 
+            /* if number of aggregators is not set we use all mpi process as aggregator*/
+            if( mThreadParams.adiosAggregators == 0 )
+               mThreadParams.adiosAggregators=mpi_size.productOfComponents();
```

Comment:
Why it is not true?
Adios 1.7.0 documentation:
```
6.1.5 MPI_AGGREGATE
The MPI_AGGREGATE method is designed to maximize write performance for large scale applications
(more than 10,000 cores) that write out data from a large subset of processors. Based upon MPI_LUSTRE
method, MPI_AGGREGATE further improves the write speed by ...
```

IMO set the default to `2` or `3` x `numOST`'s is no good default for small systems.